backend/codesirius/problems/admin.py

```
@@ -64,3 +65,83 @@
+        (_("Basic Info"), {"fields": ("problem", "s3_path", "test_count")}),
+        (
+            _("Metadata"),
+            {
+                "fields": ("created_by", "created_at", "updated_by", "updated_at"),
+                "classes": ("collapse",),
+                "description": _("Automatically tracked metadata for this record."),
+            },
+        ),
+    )
+
+    def save_model(self, request, obj, form, change):
```

Ruff (0.8.2)

Missing return type annotation for public function `save_model`

Add return type annotation: `None`

(ANN201)